orchestration/dags/import_analytics.py

```diff
@@ -286,6 +286,26 @@ def getting_service_account_token():
     dag=dag,
 )
 
+create_offer_extracted_data = BigQueryOperator(
+    task_id="create_offer_extracted_data",
+    sql=f"""Select B.offer_id, B.offer_type, json_extract(offer_extra_data, "$.author") AS Author, 
```

Comment:
C'est plus propre de mettre les mots clés SQL en majuscules
```suggestion
    sql=f"""SELECT B.offer_id, B.offer_type, json_extract(offer_extra_data, "$.author") AS Author, 
```